sample/mpp-library/src/iosMain/kotlin/com.icerockdev.library/platformLocalhost.kt

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package com.icerockdev.library
+
+actual val platformLocalhost = "0.0.0.0"
```

Comment:
it's work?

also - `com.icerockdev.library` directory is bad name - should be 3 folders `com/icerockdev/library`